src/networks/tensors.jl

```diff
@@ -116,3 +124,6 @@ const PEPOTensor{S<:ElementarySpace} = AbstractTensorMap{<:Any,S,2,4}
 Base.rotl90(t::PEPOTensor) = permute(t, ((1, 2), (4, 5, 6, 3)))
 Base.rotr90(t::PEPOTensor) = permute(t, ((1, 2), (6, 3, 4, 5)))
 Base.rot180(t::PEPOTensor) = permute(t, ((1, 2), (5, 6, 3, 4)))
+
+physicalspace(t::PEPOTensor) = space(t, 1) ← space(t, 2)'
```

Comment:
It made sense to me to make this a `HomSpace` since a PEPO tensor really maps between physcal spaces, but I can make it a `ProductSpace` if this is too confusing.

Reply:
I think I like keeping `physicalspace` like this, but adding `domain_physicalspace` and `codomain_physicalspace`. I can then also implement these for `InfinitePEPO`, which is handy for initializing purposes.

Reply:
I added `domain_physicalspace` and `codomain_physicalspace` and am quite happy with these. There's still the issue of what to do with `physicalspace` then, but resolving here in favor of the comment further below.